New Entry: Term Entry (#6385)
Codecademy/docs: content/pillow/concepts/image/terms/save/save.md

---

> Title: '.save()'
> Description: 'Saves an Image to be used.'
> Subjects:
>   - 'Code Foundations'
>   - 'Computer Science'
>   
> Tags: 
>   - 'Images'
>   - 'Libraries'
>   - 'Pillow'
> CatalogContent: 
>   - 'learn Git and GitHub'
>   - 'paths/Python'

Reviewer: ```suggestion
Title: '.save()'
Description: 'saves an image to a specified file.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags: 
  - 'Images'
  - 'Libraries'
  - 'Pillow'
CatalogContent: 
  - 'learn-python-3'
  - 'paths/data-science'
```

---

> 
> In the Pillow library, The **.save()** method is used to save an image under the given filename. This allows for the image to be saved in a recognizable way to the writer of the code. The given filename is written by the writer of the code.

Reviewer: ```suggestion

In the Pillow library, the **`.save()`** method saves an image to a specified filename. The filename, provided by the user or the code writer, determines the format in which the image is saved.
```

## Syntax

> Image.save(filename)     

Reviewer: ```suggestion
```pseudo
image.save(fp, format=None, **params)
```
```

Reviewer: refer - https://pillow.readthedocs.io/en/stable/reference/Image.html

#### Parameters:
    -**Image**: The Image object to be saved.

>     -**filename**: filename to the file to be saved.

Reviewer: Please change the parameters to the above listed syntax:

- `fp`: <description>
- `format`: <description>
- `**params`: <description>

#### RETURNS:
    none

## Example

![Image](https://unsplash.com/photos/close-up-of-a-yellow-and-blue-macaw-lylCw4zcA7I)

``` 
newImage = Image.open(r"C:\Users\Codecademy\Desktop\bird.jpg") 

newImage = newImage.save("fluffy.jpg")

> ```

Reviewer: Please:
- describe the example in one-two lines
- the image used in the example, needs to be placed in the **media** folder in docs. External links are not allowed
- the code is missing the import statement, please fix that
